### PR TITLE
Get rotation angles for computing waveform polarizations

### PR DESCRIPTION
**Pull request by vaishakp**
Get angles for computing the observer waveforms and function for transformations of spins from NR to LAL source frame.

### CODE REVIEW

File: nrcatalogtools/lvc.py [hunk 1]
```diff
@@ -183,34 +172,29 @@ def get_strain_from_lvcnr_file(path_to_file,
...
 
-    longAscNodes = 0
-    eccentricity = 0
-    meanPerAno = 0
-
-    fixed_args = [
-        distance * lal.PC_SI * 1e6, inclination, phi_ref, longAscNodes,
-        eccentricity, meanPerAno, 1 / srate
-    ]
+    fixed_args = [distance * lal.PC_SI * 1e6, 0, 0, 0, 0, 0, 1 / srate]
```
Comment (prayush) [inline nrcatalogtools/lvc.py:175]: why were these variables removed?

File: nrcatalogtools/lvc.py [hunk 2]
```diff
@@ -183,34 +172,29 @@ def get_strain_from_lvcnr_file(path_to_file,
...
+            )
         else:
             spin_args = lalsim.SimInspiralNRWaveformGetSpinsFromHDF5File(
-                f_low, Mtot, path_to_file)
+                f_low, Mtot, path_to_file
+            )
 
         mass_args = list(
-            mtotal_eta_to_mass1_mass2(Mtot * lal.MSUN_SI, h5file.attrs["eta"]))
+            mtotal_eta_to_mass1_mass2(Mtot * lal.MSUN, h5file.attrs["eta"])
```
Comment (prayush) [inline nrcatalogtools/lvc.py:196]: `MSUN_SI` it should be

File: nrcatalogtools/lvc.py [hunk 3]
```diff
@@ -107,21 +102,21 @@ def get_modes_from_lvcnr_file(path_to_file,
...
+            )
         else:
             spin_args = lalsim.SimInspiralNRWaveformGetSpinsFromHDF5File(
-                f_low, Mtot, path_to_file)
+                f_low, Mtot, path_to_file
+            )
 
-        mass_args = list(
-            mtotal_eta_to_mass1_mass2(Mtot * MSUN, h5file.attrs["eta"]))
+        mass_args = list(mtotal_eta_to_mass1_mass2(Mtot * MSUN, h5file.attrs["eta"]))
```
Comment (prayush) [inline nrcatalogtools/lvc.py:112]: where is this `MSUN` coming from? It should be `lal.MSUN_SI`.

Comment (prayush) [reply nrcatalogtools/lvc.py:112]: ... just checked the whole file - there is no definition for `MSUN`.

Comment (adivijaykumar) [reply nrcatalogtools/lvc.py:112]: `flake8` had flagged these issues as well and I have corrected them in #13.

**Pull request merged**